scripts/create_md5s.py

```diff
@@ -0,0 +1,71 @@
+import os
+from typing import Set
+
+import click
+from cpg_utils.hail_batch import get_batch, get_config, copy_common_env
+from google.cloud import storage
+
+DRIVER_IMAGE = 'australia-southeast1-docker.pkg.dev/analysis-runner/images/driver:latest'
+
+
+def create_md5s_for_files_in_directory(skip_filetypes: tuple[str, str], billing_project: str | None, gs_dir):
+    """Validate files with MD5s in the provided gs directory"""
+    b = get_batch(f'Create md5 checksums for files in {gs_dir}')
+
+    if not gs_dir.startswith('gs://'):
+        raise ValueError(f'Expected GS directory, got: {gs_dir}')
+
+    config = get_config()
+    if not billing_project:
+        billing_project = config['hail']['billing_project']
+
+    bucket_name, *components = gs_dir[5:].split('/')
+
+    client = storage.Client()
+    blobs = client.list_blobs(bucket_name, prefix='/'.join(components))
+    files: Set[str] = {f'gs://{bucket_name}/{blob.name}' for blob in blobs}
+    for obj in files:
+        if obj.endswith('.md5') or obj.endswith(skip_filetypes):
+            continue
+        if f'{obj}.md5' not in files:
```

Comment:
Do you only want to create if it doesn't exist, or add some option to force recreate?